.github/workflows/build-lint-test.yml

```diff
@@ -41,13 +35,13 @@ jobs:
       matrix:
         node-version: [18.x, 20.x]
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
-          cache: 'yarn'
-      - run: yarn --immutable
+          cache: yarn
+      - run: yarn install --immutable --immutable-cache
```

Comment:
Just for the record, I did remove it because of some caching issue, but maybe the underlying problem was not the cache itself, but more the dependencies!

- https://github.com/MetaMask/accounts/actions/runs/10900174593/job/30247230364

But let's try to re-enable it for the moment 👍